packages/metal-jsx/test/otherProps.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+import JSXComponent from '../src/JSXComponent';
+import otherProps from '../src/otherProps';
+
+describe('otherProps', function() {
+	var component;
+
+	afterEach(function() {
+		if (component) {
+			component.dispose();
+		}
+	});
+
+	it('should return object', function() {
+		class TestComponent extends JSXComponent {
+			render() {
+				return <div />;
+			}
+		}
+
+		assert.deepEqual(otherProps(new TestComponent()), {});
+	});
+
+	it('should pass through unspecified props', function() {
+		class TestComponent extends JSXComponent {
+			render() {
+				return <div />;
+			}
+		}
+
+		TestComponent.PROPS = {
+			foo: {},
+		};
+
+		assert.deepEqual(
+			otherProps(new TestComponent({baz: 'qux', foo: 'bar'})),
+			{baz: 'qux'}
+		);
+	});
+
+	it('should ignore key and ref', function() {
+		class TestComponent extends JSXComponent {
```

Comment:
I wonder if we can maybe start writing new tests completely in ES6? I mean, we are using `class` keywords already so...